main.cc

```diff
@@ -1006,13 +1007,19 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             static sharded<wasm::instance_cache> wasm_instance_cache;
             auto udf_enabled = cfg->enable_user_defined_functions() && cfg->check_experimental(db::experimental_features_t::feature::UDF);
             std::any stop_udf_cache_handlers;
+            static std::shared_ptr<wasm::alien_thread_runner> alien_runner = nullptr;
```

Comment:
You don't need the static if you have correct stop/start. The statics you see are due to bugs.

Reply:
nit, no need to initialize the `shared_ptr` with nullptr. it's already empty if created with the default ctor.

Reply:
Ok